screen_widgets.go

```diff
@@ -191,9 +191,9 @@ type Params struct {
 }
 
 type Rule struct {
-	Threshold *string `json:"threshold,omitempty"`
```

Comment:
Please use `json.Number` instead of `string` or `float64` here. I've put a long-form comment explaining why on the PR itself.

Reply:
Oh I get it. Works for me. PR updated.